### PR TITLE
update few dso outlines

### PR DESCRIPTION
**Pull request by ultrapre**
update and add few nebula's outlines.

### CODE REVIEW

File: nebulae/default/outlines.dat [hunk 1]
```diff
@@ -2963,4 +2915,407 @@
...
+07.07150 -11.07144 vertex 
+07.07481 -11.04603 vertex 
+07.07838 -10.95123 vertex 
+07.08006 -10.93613 vertex 
+07.08415 -10.86283 vertex 
+07.08584 -10.79988 vertex 
+07.08732 -10.72994 vertex 
+07.08793 -10.67084 vertex 
+07.08949 -10.57180 vertex 
+07.09133 -10.53804 end
```
Comment (alex-w) [inline nebulae/default/outlines.dat:3321]: Please fix end line issue

**Review by alex-w — CHANGES_REQUESTED** [review]
Please fix small issue

**Comment by alex-w** [comment]
NGC 6357 seems like need improve.

P.S. Please add empty line in end of file

**Comment by ultrapre** [comment]
I refer to DSS plate for NGC 6357. The picture seems to deep expose adding place.

**Comment by ultrapre** [comment]
Or it seems too Rough？

**Review by alex-w — APPROVED** [review]

**Pull request merged**